docs/dashboard-spec.md

```diff
@@ -14,6 +14,30 @@ packs/<pack_name>/dashboards/apache.json
 
 The schema adheres to the `DashboardInput` used in the [dashboardCreate](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/core-concepts/dashboards-api-migration-insights-api-nerdgraph/#dashboard-create) mutation in NerdGraph.
 
+```yaml
+####################################################################################
+# Metadata
+####################################################################################
+
+# the name of the Dashboard
+name: string, required
+
+# A short description for this Dashboard
+description: string, required
+
+# List of screenshot URLs related to this Dashboard
+type: list, required
```

Comment:
`type` is not the field, `screenshots` is. Also, this field isn't actually needed as the service just adds it.

Reply:
Will remove.